lib/hawkular/metrics/metric_api.rb

```diff
@@ -129,10 +129,13 @@ def create(definition)
 
       # Query metric definitions by tags
       # @param tags [Hash]
+      # @param options [Hash] Additional options to configure
+      # @option options [Boolean] :timestamps If timestamps should be included on the response. Defaults to true
       # @return [Array[MetricDefinition]]
-      def query(tags = nil)
+      def query(tags = nil, options = {})
```

Comment:
@pilhuhn what do you think on having a `hash` instead of a `boolean` for other params that might be added in the future to configure the query?